src/components/WebexAvatar/WebexAvatar.scss

```diff
@@ -12,6 +12,71 @@
     display: flex;
     align-items: center;
     justify-content: center;
+
+    .avatar-typing{
+      background-color: $wxc-black-alpha-50;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 100%;
+      height: 100%;
```

Comment:
remove width and height and just use position to stretch  the content on the entire parent